### PR TITLE
Add reboot host parameters to infra/roles/hosts

### PR DESCRIPTION
**Pull request by roniezr**

### CODE REVIEW

File: roles/infra/roles/hosts/tasks/main.yml [hunk 1]
```diff
@@ -37,6 +37,8 @@
     timeout: "{{ item.timeout | default(ovirt_hosts_add_timeout) }}"
     poll_interval: "{{ item.poll_interval | default(20) }}"
     hosted_engine: "{{ item.hosted_engine | default(omit) }}"
+    reboot_after_installation: "{{ reboot_added_hosts_after_installation | default(omit) }}"
+    reboot_after_upgrade: "{{ reboot_added_hosts_after_upgrade | default(omit) }}"
```
Comment (mnecas) [inline roles/infra/roles/hosts/tasks/main.yml:41]: Please update it to `item.reboot_added_hosts_after_installation` and `item.reboot_added_hosts_after_upgrade` so we can select on each host if we want to reboot it.

**Review by mnecas — COMMENTED** [review]
Thanks for the PR, just one small change is needed.

File: roles/infra/roles/hosts/README.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ The `hosts` list can contain the following parameters:
 | timeout       | 1200             | Maximum wait time for the host to be in an UP state.  |
 | poll_interval | 20               | Polling interval to check the host status. |
 | hosted_engine | UNDEF            | Specifies whether to 'deploy' or 'undeploy' hosted-engine to node. |
-| reboot_added_hosts_after_installation | UNDEF | If true reboot host after successful installation. |
-| reboot_added_hosts_after_upgrade | UNDEF | If true reboot hosts after successful upgrade. |
+| reboot_added_host_after_installation | UNDEF | If true reboot host after successful installation. |
```
Comment (kobihk) [inline roles/infra/roles/hosts/README.md:22]: maybe we can do it shorter name like: reboot_after_installation

File: roles/infra/roles/hosts/README.md [hunk 3]
```diff
@@ -19,8 +19,8 @@ The `hosts` list can contain the following parameters:
...
+| reboot_added_host_after_upgrade | UNDEF | If true reboot host after successful upgrade. |
```
Comment (kobihk) [inline roles/infra/roles/hosts/README.md:23]: maybe we can do it shorter name like: reboot_after_upgrade

**Review by kobihk — APPROVED** [review]

**Review by mnecas — APPROVED** [review]
LGTM

**Pull request merged**